internal/common/common.go

```diff
@@ -160,10 +188,21 @@ func GetConnectionNamespacedName(a *secretsv1beta1.VaultAuth) (types.NamespacedN
 		}, nil
 	}
 
-	// the VaultConnection CR must be in the same namespace as its VaultAuth.
+	// Use the NS of the AuthRef, unless it's overridden in the AuthRef Spec.
+	var vaultConnectionRefNamespace string
+	var vaultConnectionRefName string
+	vcrNames := strings.Split(a.Spec.VaultConnectionRef, "/")
+	if len(vcrNames) == 1 {
```

Comment:
similar comment as for the value parsing done for `Vault*Secrets*`, maybe consider factoring this parsing bit out to a common func.

Reply:
Refactored this into a common function in c175ed93, thanks for the suggestion, it's much cleaner